src/assets/locales/fr.json

```diff
@@ -53,13 +74,17 @@
   "button.cancel": "Annuler",
   "button.continue": "Continuer",
   "button.next": "Suivant",
+  "chat-message-input.warning-max-characters": "Caractères max",
   "chat-sidebar-container.input-placeholder.empty-room": "Il n'y a personne pour le moment...",
   "chat-sidebar-container.input-placeholder.empty-room-bot": "Envoyer le message vers {discordChannels}",
   "chat-sidebar-container.input-placeholder.occupants": "{occupantCount, plural, one {Envoyer un message à un autre...} other {Envoyer un message aux {occupantCount} autres...} }",
   "chat-sidebar-container.input-placeholder.occupants-and-bot": "{occupantCount, plural, one {Envoyer un message à un autre et {discordChannels}...} other {Envoyer un message aux {occupantCount} autres et {discordChannels}...} }",
   "chat-sidebar.input.placeholder": "Message...",
   "chat-sidebar.log-message.audio-normalization-disabled": "l'audioNormalization est désactivée.",
```

Comment:
"L'audioNormalization est désactivée."

Reply:
Ok I just tested the command in the chat. This gives:
```
la commande audioNormalization nécessite un volume de base entre 0 [aucune normalisation] et 255. 0 par défaut. La valeur recommandée est 4, si vous souhaitez activer la normalisation.il y a 1 minute

le facteur d'audioNormalization est réglé sur 4.il y a 1 minute

la commande audioNormalization nécessite un volume de base entre 0 [aucune normalisation] et 255. 0 par défaut. La valeur recommandée est 4, si vous souhaitez activer la normalisation.il y a 52 secondes

l'audioNormalization est désactivée.
```
It would be better to say "normalisation audio" when we don't talk about the command, right?
I'll put an uppercase letter for each sentence yes.

Reply:
done in https://github.com/mozilla/hubs/pull/5002/commits/066f319e79b63f2788ea890692990a5cff70d6af